src/cloud-api-adaptor/podvm/files/etc/systemd/system/confidential-data-hub.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Confidential Data Hub TTRPC API Server
-After=network.target process-user-data.service
+After=network.target cloud-final.service process-user-data.service
```

Comment:
the `cloud-final.service` seems only enabled for s390x from https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/podvm-mkosi/mkosi.presets/system/mkosi.finalize.chroot#L6

for amd64 build `cloud-final.service` is disable, so maybe we need another way to add this dependency?

Reply:
Thanks Dali, Per systemd's document, if service A defined as After= service B, it will still start if service B is disabled.